lib/train/transports/ssh_connection.rb

```diff
@@ -54,7 +54,7 @@ def initialize(options)
       @bastion_user           = @options.delete(:bastion_user)
       @bastion_port           = @options.delete(:bastion_port)
 
-      @cmd_wrapper            = CommandWrapper.load(self, @transport_options)
+      @cmd_wrapper = CommandWrapper.load(self, @transport_options)
```

Comment:
It looks relatively straightforward to me but I'll leave officially approval to someone on the core team who has had more time in train code than me. Quick question, was this a stylistic change?

Reply:
It satisfies a cop in the current ChefStyle:

```
lib/train/transports/ssh_connection.rb:57:31: C: Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.
```

Alternatively the indenting can be kept and the blank line at line 56 deleted - that also satisfies the cop.

This current form looks correct to me as this `@cmd_wrapper` isn't quite the same as the indented option passing lines above it.